### ISSUE TITLE
Very confusing, probably mistyped options on 0.61

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#12]
```
From "csshX --help":

    -y, --tile_x *number*
        (csshX only) The number of columns to use when tiling windows.

    -x, --tile_y *number*
        (csshX only) The number of rows to use when tiling windows. tile_x
        will be use if both are specified.


Shouldn't -y have a long option --tile_y instead of --tile_x?  And vice versa?


What version of the product are you using? On what operating system?

csshX 0.61 on OS X 10.5.7


```

Original issue reported on code.google.com by `nathan.s...@gmail.com` on 3 Jun 2009 at 11:56

**Comment by GoogleCodeExporter** [comment]
```
Very confusing - I agree.

Was tempted to change the implementation to match the docs... :-)

I have updated the SVN version and fixed the wiki.

Thanks for the report!
```

Original comment by `gavin.brock` on 4 Jun 2009 at 1:11
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
> Was tempted to change the implementation to match the docs... :-)

Ah, the sign of a true developer.  ;-)
```

Original comment by `nathan.s...@gmail.com` on 4 Jun 2009 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Released in csshX-0.63.tgz
```

Original comment by `gavin.brock` on 18 Jun 2009 at 3:00
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
Manually verified the change really came through.  :-)
```

Original comment by `nathan.s...@gmail.com` on 23 Jun 2009 at 9:43
